File: calculator-tr.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function ask(soru) {
  return new Promise(resolve => rl.question(soru, cevap => resolve(cevap)));
}

async function hesapMakinesi() {
  while (true) {
    console.log("----------HESAP MAKİNESİ----------");

    const a = parseFloat(await ask("İlk rakamı giriniz: "));
    const b = parseFloat(await ask("İkinci rakamı giriniz: "));
    const islem = await ask("Yapmak istedeğiniz islem türünü seçiniz (+,-,*,/): ");

    if (islem === "+") {
      console.log("Sonuc:", a + b);
    } else if (islem === "-") {
      console.log("Sonuc:", a - b);
    } else if (islem === "*") {
      console.log("Sonuc:", a * b);
    } else if (islem === "/") {
      console.log("Sonuc:", a / b);
    } else {
      console.log("Geçersiz islem");
    }

    console.log("İslem tamamlandı.\n");
  }
}

hesapMakinesi();

File: calculator-tr.java
import java.util.Scanner;

public class HesapMakinesi {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        while (true) {
            System.out.println("----------HESAP MAKİNESİ----------");

            System.out.print("İlk rakamı giriniz: ");
            int a = input.nextInt();

            System.out.print("İkinci rakamı giriniz: ");
            int b = input.nextInt();

            System.out.print("Yapmak istedeğiniz islem türünü seçiniz (+,-,*,/): ");
            String islem = input.next();

            if (islem.equals("+")) {
                System.out.println("Sonuc: " + (a + b));
            } else if (islem.equals("-")) {
                System.out.println("Sonuc: " + (a - b));
            } else if (islem.equals("*")) {
                System.out.println("Sonuc: " + (a * b));
            } else if (islem.equals("/")) {
                System.out.println("Sonuc: " + ((float)a / b));
            } else {
                System.out.println("Geçersiz islem");
            }

            System.out.println("İslem tamamlandı.\n");
        }
    }
}   
